apps/src/applab/applab.js

```diff
@@ -302,7 +306,10 @@ function queueOnTick() {
 
 function onDebugInputKeyDown(e) {
   if (e.keyCode == KeyCodes.ENTER) {
+    e.preventDefault();
     var input = e.target.textContent;
+    Applab.dc.i = Applab.dc.hist.length + 1;
+    Applab.dc.hist[Applab.dc.i - 1] = input;
```

Comment:
Can we extract these two lines into a function called something like `updateDebugConsoleHistory(commandText)`?  It may seem like overkill, but it may help make the key handler's responsibilities obvious.